cli/kontena-cli.gemspec

```diff
@@ -33,5 +33,5 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "semantic", "~> 1.5"
   spec.add_runtime_dependency "liquid", "~> 4.0.0"
   spec.add_runtime_dependency "tty-table", "~> 0.8.0"
-  spec.add_runtime_dependency "websocket-driver-kontena", "0.6.5"
+  spec.add_runtime_dependency "kontena-websocket-client", "~> 0.1.0"
```

Comment:
`s/websocket-extensions/native websocket_mask extension/`

The [`websocket-driver-ruby`](https://github.com/faye/websocket-driver-ruby/blob/master/lib/websocket/driver.rb#L23-L29) already handles the `require 'websocket_mask'` as an optional dependency with a fallback to a pure-ruby implementation... the `kontena-websocket-client` should probably depend on the pure-ruby  [`websocket-driver-kontena`](https://github.com/faye/websocket-driver-ruby/compare/master...kontena:master), and then the agent could also depend on some optional gem that provides the optional `websocket_mask` extension for better performance.

Do we have any issue regarding the details about the problems with native extensions in the CLI dependencies?